packages/react/src/components/MultiSelect/__tests__/MultiSelect-test.js

```diff
@@ -3,380 +3,443 @@
  *
  * This source code is licensed under the Apache-2.0 license found in the
  * LICENSE file in the root directory of this source tree.
+ *
  */
 
+import 'carbon-components/src/components/multi-select/_multi-select.scss';
+import { getByText, isElementVisible } from '@carbon/test-utils/dom';
```

Comment:
I think we'll want to remove the `isElementVisible` checks while we figure out the style stuff since it'll be a false positive 😞 

```suggestion
import { getByText } from '@carbon/test-utils/dom';
```